src/views/OrderView.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+    <div class="container">
+        <h1>Order varibles</h1>
+        <toast-component
+        class="toast-component mt-2"
+        v-if="toast"
+        :type="toast.type"
+        :message="toast.message"
+        @toastCloseBtnClicked="clearToast">
+      </toast-component>
+        <div class="row">
+          <div class="col-md-6">
+            <form-component
+            id="order-form"
+            :options="options"
+            :formFields="formFields"
+            :initialFormData="initialFormData"
+            :formState="formState"
+            @valueChange="onValueChanged">
+            </form-component>
+            <div>
+              <router-link
+                class="btn btn-secondary btn-outline"
+                type="button"
+                to="/"
+                tag="button">
+                Cancel
+              </router-link>
+
+              <button
+                v-if="!isSaving"
+                id="submit-btn"
+                class="btn btn-primary ml-1"
+                type="submit"
+                @click.prevent="onSubmit"
+                :disabled="formState.$invalid && formState.$touched">
+                Submit
+              </button>
+
+              <button
+                v-else
+                id="save-btn-saving"
+                class="btn btn-primary"
+                type="button"
+                disabled="disabled">
+                Submitting
+              </button>
+
+              <span v-if="!isSaving && formState.$invalid && formState.$touched" class="alert text-danger">
+                  invalid
+              </span>
+
+            </div>
+          </div>
+        </div>
+    </div>
+</template>
+
+<script>
+import Vue from 'vue'
+import { FormComponent } from '@molgenis/molgenis-ui-form'
+import { mapActions, mapMutations, mapState } from 'vuex'
+import ToastComponent from '../components/ToastComponent.vue'
+
+export default Vue.extend({
+  name: 'OrderView',
+  components: {
+    FormComponent, ToastComponent
+  },
+  data () {
+    return {
+      isSaving: false,
+      options: {
+        showEyeButton: false,
+        allowAddingOptions: false
+      },
+      formFields: [
+        {
+          type: 'text',
+          id: 'projectNumber',
+          label: 'Project number',
+          description: 'The OV number.',
+          required: () => true,
```

Comment:
More of a side note. You can still submit without the required project number if you remove it and submit quickly (before form is validated)

Reply:
Should you be able to create multiple request with the same project number?

Reply:
fixed